translator/tocwconfig/sampleConfig/complete_linux_config.yaml

```diff
@@ -119,6 +119,12 @@ processors:
             regexp: null
         initial_value: 0
         max_staleness: 0s
+    filter/1:
```

Comment:
This is not attached to any pipeline. It should filter/0 to match jmx/metrics/0. JMX needs its own pipeline for each JMX since it runs as an array all with diff filter processors.